imqsauth/http.go

```diff
@@ -619,6 +628,63 @@ func httpHandlerSetGroupRoles(central *ImqsCentral, w http.ResponseWriter, r *ht
 		central.Central.Log.Warnf("Group '%v' not found: %v", groupname, e)
 		authaus.HttpSendTxt(w, http.StatusNotFound, fmt.Sprintf("Group '%v' not found: %v", groupname, e))
 	}
+
+	// get all identities and their groups
+	identities, err := central.Central.GetAuthenticatorIdentities()
+	if err != nil {
+		central.Central.Log.Errorf("Error reading identities: %v\n", err)
+		return
+	}
+
+	// now find all identities which belong to the group in question,
+	// skipping the user that made the request
+	modifiedIdentities := make([]string, 0, 10)
+
+	for _, identity := range identities {
+		if r.token.Identity == identity {
+			// skip
+			continue
+		}
+		roles := getRolesList(central, identity)
+		if containsElement(groupname, roles) {
+			modifiedIdentities = append(modifiedIdentities, identity)
+		}
+	}
+	if len(modifiedIdentities) > 0 {
+		central.sendAll("auth:logout:" + strings.Join(modifiedIdentities, " "))
+	}
+}
+
+func containsElement(elem string, list []string) bool {
```

Comment:
It's usual to have these function parameter the other way around. ie (hackstack, needle).